src/app/views/query-runner/request/permissions/PanelList.tsx

```diff
@@ -37,6 +40,7 @@ const PanelList = ({ messages, permissions,
           searchValueChanged(event, newValue)}
         styles={{ field: { paddingLeft: 10 } }}
       />
+      <Announced message={`${permissions.length} search results available.`}/>
```

Comment:
string localization

Reply:
Since it's only a message that is being read, I tested this out with a Spanish narrator, and the narrator translates it itself